transformers/storage/cat/repository_test.go

```diff
@@ -21,11 +22,11 @@ import (
 
 var _ = Describe("Cat storage repository", func() {
 	var (
-		db           *postgres.DB
-		repo         cat.CatStorageRepository
-		fakeHeaderID int64
-		fakeAddress  = "0x12345"
-		fakeUint256  = "12345"
+		db                   *postgres.DB
```

Comment:
very minor and not an issue with this PR but maybe it's worth defining the `db` var here as `test_config.NewTestDB(test_config.NewTestNode())` here so that it's not a new connection for every test

Reply:
fdfe915